lib/std/debug_info.zig

```diff
@@ -0,0 +1,85 @@
+//! Misc utility functions for creating a `SymbolMap` for `std.debug`
+//! TODO: better name for this file?
+
```

Comment:
I can't seem to think of a better name...

Reply:
More generally, I'm not sure if throwing together all these functions in this file makes sense.